lib/components/CharCounter/CharCounter.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import clsx from 'clsx'
+import { EMPTY_STRING } from '../../consts'
+
+import './charCounter.scss'
+
+export interface CharCounterProps {
+  maxChars: number
+  messageLength: number
+  hideMaxChars?: boolean
```

Comment:
I think that we can have only one property here, in case of no `maxChars`, we can hide the div

Reply:
`maxChars` is mandatory, it's needed for showing this anyway:
<img width="644" alt="Screenshot 2024-11-14 at 14 44 02" src="https://github.com/user-attachments/assets/664edcc3-e73d-4c86-88de-4d949f56afd3">

Reply:
I think that if we are showing `0/1000 `characters, we don't need to show max :) but maybe it is only me

Reply:
I actually agree, but since we already have it in login banner, I assumed there might have been a reason, so added it conditionally